composer.json

```diff
@@ -6,7 +6,8 @@
     "minimum-stability": "dev",
     "require": {
         "drupal/condition_field": "^2.0",
-        "drupal/field_group": "~3.1"
+        "drupal/field_group": "~3.1",
+        "drupal/scheduled_transitions": "^2.1"
```

Comment:
Move scheduled transitions to [composer suggest](https://getcomposer.org/doc/04-schema.md#suggest) instead of hard dependancy (as its optional).

Reply:
I've moved it to require-dev, as it's not needed in production but is for testing, along with adding a suggest.

Reply:
@stephen-cox I should get time to test this later today, but othersiwse this looks what I expect, so we have to install scheduled transitions to test the transistions config works, but a production composer install --no-dev will not install it unless explicitly required (or localgov workflows is installed).

Reply:
[Require dev only works in root composer.json](https://getcomposer.org/doc/04-schema.md#require-dev)
I assume this is why I can't install it on a fresh site even if the default is require-dev.

Reply:
<img width="665" alt="Screenshot 2021-07-28 at 8 40 03 am" src="https://user-images.githubusercontent.com/1467480/127283496-34d16533-eb4b-4458-91ca-5e97737a7475.png">

Composer suggest works though.
> 1 package suggestions were added by new dependencies, use `composer suggest` to see details.
>
> localgovdrupal/localgov_alert_banner suggests:
>  - drupal/scheduled_transitions: Gives the ability to schedule the publishing and unpublishing of alert banners
>
> 31 additional suggestions by transitive dependencies can be shown with --all

Reply:
The require-dev does indicate that scheduled_transitions is needed for development. Specifically, some tests will fail if the module is not in the codebase. I'm not sure there's any other way to deal with this.